addons/onboarding/views/onboarding_views.xml

```diff
@@ -23,7 +23,9 @@
         <field name="arch" type="xml">
             <form>
                 <header>
-                    <button name="action_toggle_visibility" type="object" string="Toggle visibility"/>
+                    <field name="current_progress_id" invisible="1"/>
+                    <button name="action_toggle_visibility" type="object" string="Toggle visibility"
+                            attrs="{'invisible': [('current_progress_id', '=', False)]}"/>
```

Comment:
Makes sense as this action directly uses "current_progress_id".

I see that it's also available in the tree header.
I can't remember what it does, does it impact one record or multiple records? I'm guessing this one should not be conditioned the same way.

Reply:
The method will change a single `onboarding.progress` record per `onboarding.onboarding` selected in the tree view. If `current_progress_id` is `False`, it will silently do nothing.
I didn't find a simple way to make the button visible only if any of the selected record has a defined `current_progress_id`.